### PR TITLE
DOCSUP-14537: Document the log_queries_probability setting

### PR DESCRIPTION
**Pull request by sevirov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:

- Documentation for #27527

### CODE REVIEW

File: docs/en/operations/settings/settings.md [hunk 1]
```diff
@@ -3567,6 +3567,18 @@ Possible values:
...
 
+## log_queries_probability {#log-queries-probability}
+
+Allows a user to write to [query_log](../../operations/system-tables/query_log.md), [query_thread_log](../../operations/system-tables/query_thread_log.md), and [query_views_log](../../operations/system-tables/query_views_log.md) system tables only a sample of queries selected randomly with the specified probability. It helps to reduce the load with a large volume of queries in a second.
+
+Possible values:
+
+-   0 — Queries are not logged in tables.
+-   Positive floating-point number in the range [0..1]. For example, if the setting value is `0,5`, about half of the queries are logged in tables.
+-   1 — All queries are logged in tables.
```
Comment (olgarev) [inline docs/en/operations/settings/settings.md:3578]:
```suggestion
-   0 — Queries are not logged in the system tables.
-   Positive floating-point number in the range [0..1]. For example, if the setting value is `0.5`, about half of the queries are logged in the system tables.
-   1 — All queries are logged in the system tables.
```

File: docs/en/operations/system-tables/query_log.md [hunk 2]
```diff
@@ -24,6 +24,8 @@ Each query creates one or two rows in the `query_log` table, depending on the st
 2.  If an error occurred during query processing, two events with the `QueryStart` and `ExceptionWhileProcessing` types are created.
 3.  If an error occurred before launching the query, a single event with the `ExceptionBeforeStart` type is created.
 
+You can use the [log_queries_probability](../../operations/settings/settings.md#log-queries-probability) setting to set the probability with which queries are registered in the `query_log` table.
```
Comment (olgarev) [inline docs/en/operations/system-tables/query_log.md:27]:
```suggestion
You can use the [log_queries_probability](../../operations/settings/settings.md#log-queries-probability) setting to reduce the number of queries, registered in the `query_log` table.
```

File: docs/en/operations/system-tables/query_thread_log.md [hunk 3]
```diff
@@ -11,6 +11,8 @@ The flushing period of data is set in `flush_interval_milliseconds` parameter of
 
 ClickHouse does not delete data from the table automatically. See [Introduction](../../operations/system-tables/index.md#system-tables-introduction) for more details.
 
+You can use the [log_queries_probability](../../operations/settings/settings.md#log-queries-probability) setting to set the probability with which queries are registered in the `query_thread_log` table.
```
Comment (olgarev) [inline docs/en/operations/system-tables/query_thread_log.md:14]:
```suggestion
You can use the [log_queries_probability](../../operations/settings/settings.md#log-queries-probability) setting to reduce the number of queries, registered in the `query_thread_log` table.
```

File: docs/en/operations/system-tables/query_views_log.md [hunk 4]
```diff
@@ -11,6 +11,8 @@ The flushing period of data is set in `flush_interval_milliseconds` parameter of
 
 ClickHouse does not delete data from the table automatically. See [Introduction](../../operations/system-tables/index.md#system-tables-introduction) for more details.
 
+You can use the [log_queries_probability](../../operations/settings/settings.md#log-queries-probability) setting to set the probability with which queries are registered in the `query_views_log` table.
```
Comment (olgarev) [inline docs/en/operations/system-tables/query_views_log.md:14]:
```suggestion
You can use the [log_queries_probability](../../operations/settings/settings.md#log-queries-probability) setting to reduce the number of queries, registered in the `query_views_log` table.
```

**Comment by sevirov** [comment]
@evillique Please, review this PR.

File: docs/ru/operations/settings/settings.md [hunk 5]
```diff
@@ -3376,6 +3376,18 @@ SETTINGS index_granularity = 8192 │
 
 Значение по умолчанию: `1000`.
 
+## log_queries_probability {#log-queries-probability}
+
+Позволяет пользователю записать в системные таблицы [query_log](../../operations/system-tables/query_log.md), [query_thread_log](../../operations/system-tables/query_thread_log.md) и [query_views_log](../../operations/system-tables/query_views_log.md) только часть запросов, выбранных случайным образом, с указанной вероятностью. Это помогает снизить нагрузку при большом объеме запросов за секунду.
```
Comment (evillique) [inline docs/ru/operations/settings/settings.md:3381]:
```suggestion
Позволяет пользователю записывать в системные таблицы [query_log](../../operations/system-tables/query_log.md), [query_thread_log](../../operations/system-tables/query_thread_log.md) и [query_views_log](../../operations/system-tables/query_views_log.md) только часть запросов, выбранных случайным образом, с указанной вероятностью. Это помогает снизить нагрузку при большом объеме запросов в секунду.
```

File: docs/ru/operations/settings/settings.md [hunk 6]
```diff
@@ -3376,6 +3376,18 @@ SETTINGS index_granularity = 8192 │
...
+
+Возможные значения:
+
+-   0 — запросы не регистрируются в системных таблицах.
+-   Положительное число с плавающей точкой в диапазоне [0..1]. Например, при значении настройки, равной `0.5`, примерно половина запросов регистрируется в системных таблицах.
```
Comment (evillique) [inline docs/ru/operations/settings/settings.md:3386]:
```suggestion
-   Положительное число с плавающей точкой в диапазоне [0..1]. Например, при значении настройки, равном `0.5`, примерно половина запросов регистрируется в системных таблицах.
```

**Comment by sevirov** [comment]
@evillique Thank you. I fixed it.

**Pull request merged**